3scaleAdapter/pkg/threescale/threescale_test.go

```diff
@@ -147,19 +148,33 @@ func TestHandleAuthorization(t *testing.T) {
 				}
 			}
 		})
-
-		c := &Threescale{client: httpClient}
+		cacheTTl := time.Duration(time.Second * 2)
+		pc := newProxyConfigController(cacheTTl, 10)
```

Comment:
long term we should try and see what's the performance of this with the typical size useful for our purposes - (I am assuming 10 is not a representative number, and 100 probably falls too short)

Reply:
Agree, I was thinking that perhaps it would be set from a flag or the environment and whoever deploys the adapter would make it configurable. Benchmarks would help for sure